great_expectations/rule_based_profiler/domain_builder/simple_column_suffix_domain_builder.py

```diff
@@ -15,7 +14,7 @@ class SimpleColumnSuffixDomainBuilder(DomainBuilder):
 
     def __init__(
         self,
-        data_context: DataContext,
+        data_context: "DataContext",  # noqa: F821
```

Comment:
@cdkini Any idea why using the quotes was not needed before?

Reply:
Really not sure 😢 it was inevitable but I'm not 100% sure of the exact cause.

Reply:
@cdkini Could it be the config / schema objects, somehow?..  if easily preventable, it would be nice...

Reply:
I messed around with this for a bit but couldn't figure it out. Don't think it's blocking so I'm forgoing it for the time being.